### ISSUE TITLE
Run on Arch Linux fails with 'attempt to subtract with overflow' error

### ISSUE THREAD

**Issue by juanjoseamaker** [QuantumBadger/Speedy2D#10]
I was running the code of the reddit post but it failed
```rust
use speedy2d::Window;
use speedy2d::window::{WindowHandler, WindowHelper};
use speedy2d::Graphics2D;
use speedy2d::color::Color;

struct GameWindowHandler{}

impl WindowHandler for GameWindowHandler {
    fn on_draw(&mut self, helper: &mut WindowHelper, graphics: &mut Graphics2D)
    {
        graphics.clear_screen(Color::from_rgb(0.8, 0.9, 1.0));
        graphics.draw_circle((100.0, 100.0), 75.0, Color::BLUE);

        helper.request_redraw();
    }
}

fn main() {
    let window = Window::new_centered("Cube army", (640, 489)).unwrap();

    window.run_loop(GameWindowHandler{});
}
```
backtrace:
```text
thread 'main' panicked at 'attempt to subtract with overflow', /home/juanjo/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/window.rs:643:39
stack backtrace:
   0: rust_begin_unwind
             at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca/library/std/src/panicking.rs:495:5
   1: core::panicking::panic_fmt
             at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca/library/core/src/panicking.rs:92:14
   2: core::panicking::panic
             at /rustc/e1884a8e3c3e813aada8254edfa120e85bf5ffca/library/core/src/panicking.rs:50:5
   3: speedy2d::window::position_window
             at /home/juanjo/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/window.rs:643:39
   4: speedy2d::window::WindowImpl<UserEventType>::new
             at /home/juanjo/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/window.rs:403:21
   5: speedy2d::Window<UserEventType>::new_with_user_events
             at /home/juanjo/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/lib.rs:930:27
   6: speedy2d::Window::new_with_options
             at /home/juanjo/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/lib.rs:916:9
   7: speedy2d::Window::new_centered
             at /home/juanjo/.cargo/registry/src/github.com-1ecc6299db9ec823/speedy2d-1.0.2/src/lib.rs:887:9
   8: cubes_game::main
             at ./src/main.rs:19:18
   9: core::ops::function::FnOnce::call_once
             at /home/juanjo/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by QuantumBadger** [comment]
Thanks for reporting this! I've put in a fix here: https://github.com/QuantumBadger/Speedy2D/commit/494800e4a40bf139a4c9d8f46cf3cd2600df94fb

I've just published v1.0.3 with the fix in, please could you let me know if this resolves the problem for you?

**Comment by QuantumBadger** [comment]
Closing as fixed, let me know if you see the problem again.
